src/pages/resources/howtos/running-network-infrastructure.md

```
@@ -0,0 +1,178 @@
+
+* A Linux server or cloud instance
+* A container management tool that can use [OCI containers](https://opencontainers.org/) and understands docker-compose v2 files (e.g., [Docker](https://www.docker.com/) or [Podman](https://podman.io/)) (our examples will use the `docker` command)
+* TLS certificate and key files for your server's domain name stored in the server's filesystem in [PEM](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail) format. In this guide we're using certificates issued by Let's Encrypt and managed by [certbot](https://certbot.eff.org/). You can choose any other method of issuing certificates, as long as the certificates will be trusted by your users.
+
+## Create a Docker compose file
+
+Create a `docker-compose.yaml` file in an appropriate place in your server's filesystem, then open it for editing. Here we'll be storing the file in `/var/kitsune2-bootstrap` and opening it in Visual Studio Code:
+
+```bash
+sudo mkdir -p /var/kitsune2-bootstrap
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)